### ISSUE TITLE
Document how to perform a release

### ISSUE THREAD

**Issue by JLLeitschuh** [WPIRoboticsProjects/GRIP#673]
I want to put this somewhere more official but the release process is simple. Simply check out master and tag the release with git. Then upload that tag to this repository.
Travis and AppVeyor will do the rest

**Comment by nightpool** [comment]
Does appveyor make the github release automatically? Does it generate the
changelog too or is that manual?

**Comment by JLLeitschuh** [comment]
Yes, appveyor does this for Windows. And yes, the changelog is manual.
